omnibus/Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
-gem "omnibus", git: "https://github.com/chef/omnibus.git"
-gem "omnibus-software", git: "https://github.com/chef/omnibus-software.git"
+gem "omnibus", github: "chef/omnibus", branch: "ksubrama/gcc_investigate"
```

Comment:
Yes. These will be on master until we've solved the msys2 problem entirely. They have been reviewed and worked at, but there is a desire not to check them in since we think we may be able to make things work better with an omnibus-toolchain for windows.